packages/pwa/app/components/_app/index.jsx

```diff
@@ -301,8 +301,7 @@ App.getProps = async ({api}) => {
     const rootCategory = await api.shopperProducts.getCategory({
         parameters: {
             id: DEFAULT_ROOT_CATEGORY,
-            levels: DEFAULT_NAV_DEPTH,
-            locale: localeConfig.app.targetLocale
```

Comment:
This was why `parameters` already contain a locale.

Removing this line and letting our Commerce API client to auto-inject the locale and currency.

Reply:
I agree, this is an error and we shouldn't have been setting the `locale` here. It was probably an oversight of removing the locale param when moving to a "global" config pattern. But do you think it's wise that we always just ignore the `locale` parameter? 

I can envision a scenario where you will still want to honour this property and simply treat the globally set value as the fallback. The issue we'll have now is that people will look at the api docs on how to use a given endpoint and it will say that it accepts local, but we'll just ignore it and instead use the globally set value on the client object. 

This will have some knock on effects with the logic that you added to remove those values via destructuring. 

Thoughts?

Reply:
> I can envision a scenario where you will still want to honour this property and simply treat the globally set value as the fallback.

Ok let's support that scenario. We can have this logic then in our commerce-api client:

```js
                                // By default we send the locale param and don't send the currency param.
                                const {sendLocale = true, sendCurrency = false} = apiConfigs[key]

                                const {
                                    locale: overrideLocale,
                                    currency: overrideCurrency,
                                    ...rest
                                } = fetchOptions.parameters || {}

                                const sendLocaleForCurrentProp =
                                    !!overrideLocale ||
                                    (Array.isArray(sendLocale)
                                        ? sendLocale.includes(prop)
                                        : !!sendLocale)

                                const sendCurrencyForCurrentProp =
                                    !!overrideCurrency ||
                                    (Array.isArray(sendCurrency)
                                        ? sendCurrency.includes(prop)
                                        : !!sendCurrency)

                                fetchOptions.parameters = {
                                    ...rest,
                                    ...(sendLocaleForCurrentProp
                                        ? {locale: overrideLocale || locale}
                                        : {}),
                                    ...(sendCurrencyForCurrentProp
                                        ? {currency: overrideCurrency || currency}
                                        : {})
                                }

```

Reply:
Maybe a even simpler variable name would be `includeGlobalLocale` and `includeGlobalCurrency` ?